File: XWeather/XWeather/Domain/Wu/WuAcResults.cs
using System.Collections.Generic;

namespace XWeather.Domain
{
	public class WuAcResults
	{
		public List<WuAcLocation> Results { get; set; }
	}

	public class WuAcLocation
	{
		public string name { get; set; }
		public string type { get; set; }
		public string c { get; set; }
		public string zmw { get; set; }
		public string tz { get; set; }
		public string tzs { get; set; }
		public string l { get; set; }
		public string ll { get; set; }
		public double lat { get; set; }
		public double lon { get; set; }

		public bool Current { get; set; }
		public bool Selected { get; set; }
	}
}

File: Swift/XWeather/XWeather/Domain/Wu/WuAcResults.swift
//
//  WuAcResults.swift
//  XWeather
//
//  Created by Colby Williams on 9/13/16.
//  Copyright © 2016 Colby Williams. All rights reserved.
//

import Foundation

public class WuAcResults {

	public var Results: [WuAcLocation]?
}

public class WuAcLocation {

	public var name: String?
	public var type: String?
	public var c: String?
	public var zmw: String?
	public var tz: String?
	public var tzs: String?
	public var l: String?
	public var ll: String?
	public var lat: Double?
	public var lon: Double?

	public var Current: Bool = false
	public var Selected: Bool = false
}
